diagnostic_msgs/srv/AddDiagnostics.srv

```diff
@@ -0,0 +1,28 @@
+# This service is used as a request to create a bond between the diagnostic
+# aggregator and the node sending the request.
+
```

Comment:
This could be misleading. As it is, it seems like you could load some params and then call this service on the command line to do the job.

The reality however, is you need a loader script/program that is responsible for bonding with *and* transferring information via this service to the aggregator, such as the python loader you have contributed. Could you reword this to that effect, perhaps with a link to the tutorial for adding diagnostics at runtime?